src/main/java/hudson/plugins/mercurial/MercurialSCM.java

```diff
@@ -927,7 +950,9 @@ private boolean causedByMissingHg(IOException e) {
                 listener.error("Failed to use repository cache for " + getSource(env));
                 return null;
             }
-        } catch (Exception x) {
+        } catch (Error error) {
+            throw error;
+        } catch (Throwable x) {
```

Comment:
This is essentially the same as catching `Exception`, except harder to read. What was wrong with the original code here, anyway? Maybe I am missing something. Sounds like FindBugs finding non-bugs.